src/spec-configuration/containerTemplatesOCI.ts

```diff
@@ -35,3 +56,11 @@ async function fetchOCITemplateManifestIfExistsFromUserIdentifier(output: Log, e
 	const templateRef = getRef(output, identifier);
 	return await fetchOCIManifestIfExists(output, env, templateRef, manifestDigest, authToken);
 }
+
+function replaceTemplatedValues(output: Log, template: string, options: TemplateOptions) {
+	const pattern = /\${templateOption:\s*(\w+?)\s*}/g; // ${templateOption:XXXX}
+	return template.replace(pattern, (_, token) => {
+		output.write(`Replacing ${token} with ${options[token]}`);
+		return options[token] || '';
```

Comment:
The options's value may contain escape chars (eg - `nodeVersion: "lts/*"`).
Would this function handle that or the should the options be passed with escaped value?

Reply:
I think this regex is ok? I've updated the test to add in `lts/*` to validate your example.  Is there another example you can think of that would be good to test?

Reply:
looks good. thanks!